extensions/amp-story/0.1/amp-story-consent.js

```diff
@@ -168,18 +171,21 @@ export class AmpStoryConsent extends AMP.BaseElement {
           TAG, 'Expected "publisher-logo-src" attribute on <amp-story>');
     }
 
-    const storyConsentConfig =
-        this.consentConfig_ && this.consentConfig_['story-consent'];
     const consentId = Object.keys(this.consentConfig_.consents)[0];
-    this.storyConsentEl_ = renderAsElement(
-        this.win.document, getTemplate(storyConsentConfig, consentId, logoSrc));
-    createShadowRootWithStyle(this.element, this.storyConsentEl_, CSS);
 
-    // Allow <amp-consent> actions in STAMP (defaults to no actions allowed).
-    this.actions_.addToWhitelist('AMP-CONSENT.accept');
-    this.actions_.addToWhitelist('AMP-CONSENT.reject');
+    // Story consent config is set by the `assertAndParseConfig_` method.
+    if (this.storyConsentConfig_) {
```

Comment:
nit: since you checked with `user().assert()` I don't think this is necessary.

Reply:
talked offline. did this to pass linter check. 
@gmajoulet You may want to add details here : )

Reply:
For some reason the linter didn't accept a ``user().assert`` or a ``/** @type {!Object} (...) */``...